markdown-link-check

```diff
@@ -43,6 +43,7 @@ function getInputs() {
         .option('-v, --verbose', 'displays detailed error information')
         .option('-a, --alive <code>', 'comma separated list of HTTP codes to be considered as alive', commaSeparatedCodesList)
         .option('-r, --retry', 'retry after the duration indicated in \'retry-after\' header when HTTP code is 429')
+        .option('--projectBaseUrl <url>', 'the URL to use for {{BASEURL}} replacement')
```

Comment:
Note that this would be the only CLI arg without a shorthand option (`-p` was already taken for `--progress`).